modules/webbrowser/src/webbrowserclient.cpp

```diff
@@ -227,4 +226,41 @@ void WebBrowserClient::OnLoadError(CefRefPtr<CefBrowser> browser, CefRefPtr<CefF
     frame->LoadURL(WebBrowserModule::getDataURI(ss.str(), "text/html"));
 }
 
+bool WebBrowserClient::OnConsoleMessage(CefRefPtr<CefBrowser> browser, cef_log_severity_t level,
+                                        const CefString& message, const CefString& source,
+                                        int line) {
+
+    if (auto lc = LogCentral::getPtr()) {
+        std::string src = source.ToString();
+
+        std::string file = "";
+        if (src.rfind("file://", 0) == 0) {
+            replaceInString(src, "\\", "/");
+            file = splitString(src, '/').back();
+        }
+
+        LogLevel loglevel;
+        switch (level) {
+            case LOGSEVERITY_DISABLE:
+                return false;
+            case LOGSEVERITY_ERROR:
+                loglevel = LogLevel::Warn;
+                break;
+            case LOGSEVERITY_WARNING:
+                loglevel = LogLevel::Warn;
+                break;
+            case cef_log_severity_t::LOGSEVERITY_DEBUG:
+            case cef_log_severity_t::LOGSEVERITY_INFO:
+            case cef_log_severity_t::LOGSEVERITY_DEFAULT:
+            default:
+                loglevel = LogLevel::Info;
+                break;
+        }
+
+        lc->log("WebBrowserClient", loglevel, LogAudience::Developer, file.c_str(), "", line,
+                message.ToString());
```

Comment:
return true afterwards since we handle the logging?

Reply:
fixed